diff --git i/t/t1517-outside-repo.sh w/t/t1517-outside-repo.sh
index c824c1a25c..b5194552ab 100755
--- i/t/t1517-outside-repo.sh
+++ w/t/t1517-outside-repo.sh
@@ -118,10 +118,10 @@ do
 	gui | gui--askpass | \
 	http-backend | http-fetch | http-push | init-db | \
 	merge-octopus | merge-one-file | merge-resolve | mergetool | \
-	mktag | p4 | p4.py | pickaxe | remote-ftp | remote-ftps | \
+	mktag | pack-redundant | p4 | p4.py | pickaxe | remote-ftp | remote-ftps | \
 	remote-http | remote-https | replay | send-email | \
 	sh-i18n--envsubst | shell | show | stage | submodule | svn | \
-	upload-archive--writer | upload-pack | web--browse | whatchanged)
+	upload-archive--writer | web--browse | whatchanged)
 		expect_outcome=expect_failure ;;
 	*)
 		expect_outcome=expect_success ;;
