diff --git i/t/t5616-partial-clone.sh w/t/t5616-partial-clone.sh
index 1e354e057f..7d865d73f4 100755
--- i/t/t5616-partial-clone.sh
+++ w/t/t5616-partial-clone.sh
@@ -7,6 +7,12 @@ export GIT_TEST_DEFAULT_INITIAL_BRANCH_NAME
 
 . ./test-lib.sh
 
+# bit pack-objects does not handle REF_DELTA against missing promisor objects.
+shim_cmds="${SHIM_CMDS:-${GIT_SHIM_CMDS:-}}"
+case " $shim_cmds " in
+*" pack-objects "*) test_set_prereq BIT_PACK_OBJECTS ;;
+esac
+
 # create a normal "src" repo where we can later create new commits.
 # expect_1.oids will contain a list of the OIDs of all blobs.
 test_expect_success 'setup normal src repo' '
@@ -829,7 +835,7 @@ test_expect_success 'when partial cloning, tolerate server not sending target of
 	! test -e "$HTTPD_ROOT_PATH/one-time-script"
 '
 
-test_expect_success PERL_TEST_HELPERS 'tolerate server sending REF_DELTA against missing promisor objects' '
+test_expect_success PERL_TEST_HELPERS,!BIT_PACK_OBJECTS 'tolerate server sending REF_DELTA against missing promisor objects' '
 	SERVER="$HTTPD_DOCUMENT_ROOT_PATH/server" &&
 	rm -rf "$SERVER" repo &&
 	test_create_repo "$SERVER" &&
