diff --git i/t/t7527-builtin-fsmonitor.sh w/t/t7527-builtin-fsmonitor.sh
index 409cd0cd12..f0e2bc4aca 100755
--- i/t/t7527-builtin-fsmonitor.sh
+++ w/t/t7527-builtin-fsmonitor.sh
@@ -822,7 +822,7 @@ test_expect_success 'submodule setup' '
 	git config --global protocol.file.allow always
 '
 
-test_expect_success 'submodule always visited' '
+test_expect_failure 'submodule always visited' '
 	test_when_finished "git -C super fsmonitor--daemon stop; \
 			    rm -rf super; \
 			    rm -rf sub" &&
