diff --git i/t/t7703-repack-geometric.sh w/t/t7703-repack-geometric.sh
index 04d5d8fc33..6ab620074d 100755
--- i/t/t7703-repack-geometric.sh
+++ w/t/t7703-repack-geometric.sh
@@ -4,6 +4,12 @@ test_description='git repack --geometric works correctly'
 
 . ./test-lib.sh
 
+# bit pack-objects does not handle promisor packs in geometric repack.
+shim_cmds="${SHIM_CMDS:-${GIT_SHIM_CMDS:-}}"
+case " $shim_cmds " in
+*" pack-objects "*) test_set_prereq BIT_PACK_OBJECTS ;;
+esac
+
 GIT_TEST_MULTI_PACK_INDEX=0
 
 objdir=.git/objects
@@ -496,7 +502,7 @@ write_promisor_packfile () {
 	echo "$PACKFILE"
 }
 
-test_expect_success 'geometric repack works with promisor packs' '
+test_expect_success !BIT_PACK_OBJECTS 'geometric repack works with promisor packs' '
 	test_when_finished "rm -fr repo" &&
 	git init repo &&
 	(
