diff --git i/t/t7703-repack-geometric.sh w/t/t7703-repack-geometric.sh
index 04d5d8fc33..1886852cf0 100755
--- i/t/t7703-repack-geometric.sh
+++ w/t/t7703-repack-geometric.sh
@@ -496,7 +496,7 @@ write_promisor_packfile () {
 	echo "$PACKFILE"
 }
 
-test_expect_success 'geometric repack works with promisor packs' '
+test_expect_failure 'geometric repack works with promisor packs' '
 	test_when_finished "rm -fr repo" &&
 	git init repo &&
 	(
